### DISCUSSION TITLE
Enable Two languages in single apk

### DISCUSSION THREAD

**Discussion by UsmanRepos** [AnySoftKeyboard/AnySoftKeyboard#3281] [Q&A] [answered]
Greetings,

I need to make one apk having two languages i.e. English and Arabic or French. Can someone guide to setup the project.

thanks.

**Comment by menny** [comment, marked as answer]
It is as simple as adding the desired pack as an additional dependency.

For example, English is added to the APK by depending on the English pack module: https://github.com/AnySoftKeyboard/AnySoftKeyboard/blob/main/ime/app/build.gradle#L95

Add another dependency pointing to the pack(s) you want in your APK, and build.

**Reply by UsmanRepos** [reply]
Thanks ...
Its work .

**Reply by UsmanRepos** [reply]
Greeting,
When we enable two Keyboards (For Example, English and Arabic) and switch from English to Arabic then almost all things change but Speech to Text Feature is not change. It convert the speech in English Not in Arabic. Can Guide Why ?
Thanks.
